docs/release-notes/cloud/cloud-2021-11.md

```diff
@@ -29,7 +29,7 @@ These release notes are for the Codacy Cloud updates during November 2021.
 -   Fixed an issue that prevented the message "Refresh the page to see the results" from being displayed on the commit and pull request pages after an analysis was completed. (CY-5187)
 -   Fixed some default regular expressions on [<span class="skip-vale">codacy-checkstyle</span>](https://github.com/codacy/codacy-checkstyle){: target="_blank"} that could cause the code pattern PackageName to report false positives when configured using the Codacy UI. (CY-5185)
 -   Fixed an issue that caused ESLint to report false positive issues regarding import statement organization if prettier was enabled. (CY-5143)
--   Fixed the issue cards in file lists that didn't display the issue category. (CY-5125)
+-   Fixed the issue cards in the context of a file to display the issue category. (CY-5125)
 
```

Comment:
```suggestion
- Fixed an issue that sporadically caused repository quality settings to only be saved partially. (CY-5380)

```

Reply:
this came from a bug that was detected during the release process (we also created a bug and release note for it so it can be included in future cloud release notes)